components/ui/AppMultiSelect.vue

```diff
@@ -23,10 +23,10 @@ type multiSelectOption = {
 }
 
 @Component
-export default class extends Vue {
+export default class AppMultiSelect extends Vue {
   @Prop(Array) options!: Array<string>
-  @Prop(String) label!: any
-  @Prop(Array) value!: any
+  @Prop(String) label!: string
+  @Prop(Array) value!: string[]
```

Comment:
I'm wondering if values will only ever consist of strings... still, I think this is good for now

Reply:
I particularly like the approach of first defining the types for the current use-case and adjusting the definition later if needed.

I think that TypeScript types are handier the more strict one is.